models/system/bert-base-cased/spec.yaml

```diff
@@ -6,6 +6,9 @@ properties:
   datasets: bookcorpus, wikipedia
   finetuning-tasks: text-classification, token-classification, question-answering
   languages: en
+  recommended-sku: Standard_ND40RS_v2
```

Comment:
Are `recommended-sku`, `minimum-sku` specific to finetune? Why not add a finetune prefix to it as well? And should these be added to tags? There is `min_inference_sku` already in tags. They all should be at similar places imo either properties or tags.

Reply:
@novaturient95 Do you know muhammad's github id; we can add him as a reviewer to this PR?